doc/baremetal/gcc-mcpu.rst

```diff
@@ -161,4 +186,62 @@ values for ARC 600 and ARC 700.
     arc601_mul32x16      Y                                 -mmul32x16
    ================= ======== ======== ================== ============
 
+
+ARCv3 Toolchain
+---------------
+
+The next ``-mcpu`` options are recognized by the ARC64 GCC compiler.
+
+.. table:: -mcpu variants for ARCv3
+
+   ======== ======== ======== ========
+    -mcpu    -msimd   -m128    -mll64
+   ======== ======== ======== ========
+   hs5x       Y       N.A.      N
+   hs58       Y       N.A.      Y
```

Comment:
Doesn't `hs58` include "advanced mpy9 SIMD instructions" compared to `hs5x`?

Reply:
That should be under `-msimd` option. Do you suggest to have `hs5x`  to be without `-msimd` option?

Reply:
Well, I somehow understood that incorrectly. And so I suggest to keep settings as they are now. Sorry for the noise.